k8s/k8s.go

```diff
@@ -1008,6 +1014,22 @@ func (k *k8sOps) GetDeploymentsUsingStorageClass(scName string) ([]apps_api.Depl
 
 // DaemonSet APIs - BEGIN
 
+func (k *k8sOps) CreateDaemonSet(ds *apps_api.DaemonSet) (*apps_api.DaemonSet, error) {
+	if err := k.initK8sClient(); err != nil {
+		return nil, err
+	}
+
+	return k.client.Apps().DaemonSets(ds.Namespace).Create(ds)
+}
+
+func (k *k8sOps) ListDaemonSets(namespace string, listOpts meta_v1.ListOptions) (*apps_api.DaemonSetList, error) {
```

Comment:
nit: in most of sched ops calls we return a slice of Objects instead of the k8s List object. should we send the same here so callers can just iterate on it instead of worrying about the metadata?

Reply:
Yes, returning list is better. Done in incremental.